@types/node 7.0.30 (latest version) breaks build

The latest version of @types/node directory, which is version 7.0.30, breaks the build. Throws this error message: 
test/client.ts(651,27): error TS2345: Argument of type '"uncaughtException"' is not assignable to parameter of type 'Signals'.
test/client.ts(705,27): error TS2345: Argument of type '"uncaughtException"' is not assignable to parameter of type 'Signals'.

--- a/package.json
+++ b/package.json
@@ -64,7 +64,7 @@
     "@types/chai-as-promised": "0.0.31",
     "@types/lodash": "^4.14.62",
     "@types/mocha": "^2.2.31",
-    "@types/node": "^7.0.5",
+    "@types/node": "7.0.29",
     "@types/promises-a-plus": "0.0.27",
     "@types/sinon": "^2.1.0",
     "benchmark": "^2.1.3",

